src/libs/routes/permission.ts

```diff
@@ -0,0 +1,16 @@
+import { permissions } from './constant';
+export default function hasPermission(moduleName: string, role: string, permissionType: string): boolean {
```

Comment:
use es6

Reply:
Updated the function name and code according to the ES6